changelog/17.feature.rst

```diff
@@ -0,0 +1 @@
+Added a new feature ```fourier_normalizing_radial_gradient_filter``` to normalize the radial brightness gradient using a fourier approximation.
```

Comment:
`fourier_normalizing_radial_gradient_filter` should be a full sphinx interlink.

Reply:
I hope it is fixed.